featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1611,7 +1611,7 @@ def test_some_instances_not_in_data(pd_es):
 
     assert all(fm.index.values == cutoff_time["instance_id"].values)
     for x, y in zip(fm.columns, [ifeat_answer, prop_answer, dfeat_answer]):
-        np.testing.assert_array_equal(fm[x], y)
+        np.testing.assert_array_equal(fm[x].astype('float64'), y)
```

Comment:
I'm on the fence about this change. We now get different results in our feature matrix depending on what version of pandas we use - this seems like a possible problem from a user perspective, no?

Here we are making the test pass, by type casting the result, but I think if we expect our feature matrix result to be different, we should change our expected answer `y`, not type cast the feature matrix.

Based on the notes in the related pandas issue, I'm not sure if this will be a change that will remain long term, and this is something we might want to revert in the future if new versions of pandas reimplement the old concat behavior.

Reply:
@thehomebrewnerd The issue that caused this change will be fixed in pandas 1.3.1, I believe. https://github.com/pandas-dev/pandas/issues/42092

If we want to raise our min pandas version again, we can remove this dtype change